lib/content-api.js

```diff
@@ -60,6 +66,21 @@ function transformData(obj, dirpath) {
 }
 
 ContentApi.prototype.getRecord = function getRecord(slug) {
+  return new Promise((resolve) => {
+    this
+      .getRecordFromApi(slug)
```

Comment:
Looks a bit odd. I'd suggest `this.getRecordFromApi(slug)` followed by chain methods.